.github/workflows/ci.yml

```diff
@@ -49,8 +49,10 @@ jobs:
 
       - name: lint
         run: |
-          make lint base=origin/${{github.base_ref}}
+          make lint base=origin/${BASE_REF}
           sudo chown -R $(id -u):$(id -g) .cache/golangci-lint  # needed to archive cache
+        env:
+          BASE_REF: ${{ github.base_ref }}
```

Comment:
Would it be worth adding a comment, to prevent future work from reintroducing the old pattern?

Reply:
added a comment